data/Application-Dark.css

```diff
@@ -0,0 +1,167 @@
+.roundy-label,
+.roundy-label:hover,
+.roundy-label:selected,
+.roundy-label:selected:focus,
+.roundy-label:hover:selected {
+    background-image: none;
+    background-color: @BLACK_300;
+    box-shadow: none;
+    color: alpha(@text_color, 0.7);
+    font-weight: 700;
+    border-radius: 10px;
+    padding: 0 6px;
+    margin: 0 3px;
+    border-width: 0;
+}
+
+.green {
+    background-color: alpha(@LIME_300, 0.6);
+}
+
+.blue {
+    background-color: alpha(@BLUEBERRY_300, 0.6);
+}
+
+/* .vertical-label,
```

Comment:
If you are not using these styles, might as well remove them. Same for other places where it's commented.